vendor/github.com/ethereum/go-ethereum/common/message/message.go

```diff
@@ -27,7 +27,7 @@ type Status int
 
 // consts of all message delivery status.
 const (
-	PendingStatus Status = iota << 3
+	PendingStatus Status = iota + 10
```

Comment:
I guess it's a typo - +1 instead of +10 here? :)

Reply:
I consider, we should have the value of Direction and Status start at different int points, but do you think sticking to one is fine?

Reply:
Yeah, it's totally fine as they belong to different types. Compiler won't permit code like `message.Status == message.Direction`, right?

Reply:
True, will make the change, currently making things more DRY.